### PR TITLE
Gsoc18 commits for volunteer and authentication apps

### PR DESCRIPTION
**Pull request by anjali-dhanuka**
# Description
Pull commits for Volunteer and Authentication apps

# Type of Change
- Code
- Quality Assurance
- User Interface
- Outreach
- Documentation

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

# How Has This Been Tested?
Describe the tests you ran to verify your changes. Provide instructions or GIFs so we can reproduce. List any relevant details for your test.

# Checklist:
**Delete irrelevant options.**

- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged 

**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

### CODE REVIEW

**Comment by kamsuri** [comment]
@anjali-dhanuka can you please update the "how has thisbeen tested?" section.

File: vms/volunteer/tests/test_volunteerReport.py [hunk 1]
```diff
@@ -94,15 +134,24 @@ def test_date_field(self):
         register_shift_utility()
         log_hours_utility()
 
-        report_page.login_and_navigate_to_report_page(self.live_server_url)
-        report_page.fill_report_form({ 'start' : '2015-06-11', 'end' : '2017-06-16'})
-        self.verify_shift_details('1','3.0')
+        report_page.navigate_to_report_page()
+        report_page.fill_report_form({
+            'start': '2048-06-11',
```
Comment (kamsuri) [inline vms/volunteer/tests/test_volunteerReport.py]: Any specific reason for these dates?

Comment (nida) [reply vms/volunteer/tests/test_volunteerReport.py]: 2048 ? start and end is 2050?

**Comment by Monal5031** [comment]
@nida This PR was to merge infra commits into code these changes are from infra branch (not done by @anjali-dhanuka ) btw these are already merged in gsoc18-code in a separate PR

**Comment by nida** [comment]
@Monal5031  Thanks. @anjali-dhanuka can you please track and update the PRs.

**Pull request closed without merging**